Automattic/studio
# Studio: Make About Studio window more compelling

Related to https://github.com/Automattic/dotcom-forge/issues/6853

## Proposed Changes

This PR adds a custom `About Studio` window to the Studio app:

<img width="283" alt="Capture d’écran, le 2024-05-13 à 14 59 27" src="https://github.com/Automattic/studio/assets/25575134/1f682c73-6531-4d02-8b5a-04d83d35ecc4">

Figma design RToz6tIuQ7nlZrikBte4GU-fi-2762_82944

* Please note that I have omitted the version number in `( )` after [discussing with Matt ](https://github.com/Automattic/dotcom-forge/issues/6853#issuecomment-2107079386)as it seems redundant. 

## Testing Instructions

* Start the app with `nvm use && npm install && npm start`
* On Studio window
* On the top, in the toolbar, click on `Electron -> About Studio`
* Confirm that you see the window as on the screenshot
* Confirm that you can click on the links and they work as expected
* Confirm that `WordPress.com hosting` link has tracking URL: https://wordpress.com/?utm_source=studio&utm_medium=referral&utm_campaign=about_screen
* Test other menu options and confirm everything works as expected

**Notes**:

* I only tested this approach on MacOs. I would be good to run a test on Windows or Linux to see how it looks
* You might also see the small flash while the content is loading. I noticed it as well and but I have also noticed it in other windows such as Settings 

## Pre-merge Checklist
